invokeai/frontend/web/src/features/nodes/util/graphBuilders/buildLinearImageToImageGraph.ts

```diff
@@ -363,6 +364,11 @@ export const buildLinearImageToImageGraph = (
   // Add IP Adapter
   addIPAdapterToLinearGraph(state, graph, DENOISE_LATENTS);
 
+  // High resolution fix.
+  if (state.generation.hrfToggled) {
+    addHrfToGraph(state, graph);
+  }
+
```

Comment:
Isn't img2img kinda already HRF? I guess this is adding an intermediary step? Tbh I'm having trouble wrapping my head around it and can't remember if we did this on img2img before

Reply:
afiak img2img doesn't provide a rescale latents functionality. The ERSGAN method is doable with via a couple invokations in the img2img linear flow.

Reply:
It does resize the image to the output size first, but it does so with a simple raster image resize (which typically produces better results than rescaling the latents, but worse than ESRGAN or another AI-supported upscaler).

Reply:
given the new HRF initial resolution sliders I've removed this from img2img